src/qt/trafficgraphwidget.cpp

```diff
@@ -102,74 +114,52 @@ void TrafficGraphWidget::paintEvent(QPaintEvent *)
         }
     }
 
-    if(!vSamplesIn.empty()) {
-        QPainterPath p;
-        paintPath(p, vSamplesIn);
-        painter.fillPath(p, QColor(0, 255, 0, 128));
+    const TrafficGraphData::SampleQueue& queue = trafficGraphData.getCurrentRangeQueue();
```

Comment:
I belive spacing is correct here:
const TrafficGraphData::SampleQueue& queue = trafficGraphData.getCurrentRangeQueue();
is inside TrafficGraphWidget::paintEvent method.